ui/components/component-library/avatar-account/avatar-account.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { BaseAvatar } from '../base-avatar';
+
+import { SIZES } from '../../../helpers/constants/design-system';
+
+export const diameters = {
+  xs: '16',
+  sm: '24',
+  md: '32',
+  lg: '40',
+  xl: '48',
+};
+
+export const AvatarAccount = ({ children, size, className, ...props }) => {
+  return (
+    <BaseAvatar
+      size={size}
+      className={classnames('avatar-account', className)}
+      {...props}
+    >
+      {children}
```

Comment:
I think instead of making the engineer pass in the different options we can add a `type` prop and do the logic within the component?